app/models/podcast_import.rb

```diff
@@ -1,379 +1,61 @@
-require "loofah"
-require "hash_serializer"
-
 class PodcastImport < ApplicationRecord
   include ImportUtils
 
-  serialize :config, HashSerializer
-
-  attr_accessor :feed, :feed_raw_doc, :templates, :feed_source
+  store :config, coder: JSON
 
   belongs_to :podcast, -> { with_deleted }, touch: true, optional: true, autosave: true
-  has_many :episode_imports, -> { where(has_duplicate_guid: false).includes(:podcast_import) }, dependent: :destroy
+  has_many :episode_imports, dependent: :destroy
 
   before_validation :set_defaults, on: :create
 
-  validates :url, presence: true
-
   enum :status, {
-    audio_saved: AUDIO_SAVED,
-    complete: COMPLETE,
     created: CREATED,
-    saved: SAVED,
-    failed: FAILED,
-    feed_retrieved: FEED_RETRIEVED,
+    started: STARTED,
     importing: IMPORTING,
-    retrying: RETRYING,
-    started: STARTED
+    complete: COMPLETE,
+    error: ERROR
   }, prefix: true
```

Comment:
Again, setting up a simple structure for how the base classes work.  The rest of this code migrated to the `podcast_rss_import.rb` class.